.github/workflows/pr.yml

```diff
@@ -7,6 +7,9 @@ concurrency:
   group: ${{ github.ref }}-${{ github.workflow }}
   cancel-in-progress: true
 on:
+  schedule:
+#   Run on end of the day
+    - cron: '0 0 * * *'
```

Comment:
Why was this added? Is it meant to run a build at least once a day?

Reply:
yes, to run a daily build.